src/main/cpp/t02_star.cpp

```diff
@@ -24,6 +24,21 @@
 using namespace std;
 
 int t02_star() {
+    int n;
+    cin >> n;
+    char a[n][n];
+
+    for (int i = 0; i < n; i++) {
+        for (int j = 0; j < n; j++) {
+            if ((i == j) || (i == n % 2) || (j == n % 2) || (i == (n - j + 1))) {
+                a[i][j] = "*";
+            }
+            else {
+                a[i][j] = ".";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assigning a pointer to an integer is not portable.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4568488)